psutil/_pssunos.py

```diff
@@ -394,7 +398,10 @@ def _proc_basic_info(self):
 
     @memoize_when_activated
     def _proc_cred(self):
-        return cext.proc_cred(self.pid, self._procfs_path)
+        @wrap_exceptions
```

Comment:
Why not decorate the method instead?

Reply:
I tried stacking the `@wrap_exceptions` on top of `@memoize_when_activated` of `_proc_cred` but it didn't work. Something about a missing attribute, IIRC.